packages/react-variant-view/src/model/Mutation.ts

```diff
@@ -0,0 +1,37 @@
+// this component comes from signal
```

Comment:
We should move this to `model/signal` under `cbioportal-utils`.

Also maybe directly use `SignalMutation` coming from Genome Nexus instead of `IMutation`, and rename `IExtendedMutation` to `IExtendedSignalMutation`.

Reply:
Good point, `ICountByTumorType` could be replaced by `CountByTumorType` from Genome Nexus also.

Reply:
Right, we can use the existing model from genome nexus directly. I guess this is also true for `SignalMutation`. We can always go over what we have and clean up duplicate models later though.